sdk/azurestackhci/Azure.ResourceManager.Hci/src/Generated/PublisherData.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using Azure.Core;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.Hci
+{
+    /// <summary> A class representing the Publisher data model. </summary>
+    public partial class PublisherData : ResourceData
+    {
+        /// <summary> Initializes a new instance of PublisherData. </summary>
+        public PublisherData()
+        {
+        }
+
+        /// <summary> Initializes a new instance of PublisherData. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="provisioningState"> Provisioning State. </param>
+        internal PublisherData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string provisioningState) : base(id, name, resourceType, systemData)
+        {
+            ProvisioningState = provisioningState;
+        }
+
+        /// <summary> Provisioning State. </summary>
+        public string ProvisioningState { get; }
```

Comment:
Should this be of type [HciProvisioningState.cs](https://github.com/Azure/azure-sdk-for-net/pull/33589/files#diff-226260d0b2f6f1106c429b82da43246a2ce629877d53bba0f587db4a34c9705c)?

Reply:
No this will be Provisioning state.